### ISSUE TITLE
run not work!

### ISSUE THREAD

**Issue by keyiis** [nwjs-community/nw-builder#442]
OS:win10 x64
nw:0.14.7
When I execute the following command,nothing happen,no error,no runing: 
`.\node_modules\.bin\nwbuild -p win32 -v 0.14.7 --cacheDir .\nw-cache -r output\build`
I already download sdk in  `.\nw-cache`
![qq 20170417121950](https://cloud.githubusercontent.com/assets/7766085/25078727/71b7242a-2368-11e7-9fe8-72a5f8a2e3ea.png)
below is app script floder:
![qq 20170417122043](https://cloud.githubusercontent.com/assets/7766085/25078736/8b366546-2368-11e7-97ab-6fa8ab8efa77.png)

when wait longtime,error output:
```
{ Error: connect ETIMEDOUT 107.170.237.181:443
    at Object.exports._errnoException (util.js:1022:11)
    at exports._exceptionWithHostPort (util.js:1045:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1087:14)
  code: 'ETIMEDOUT',
  errno: 'ETIMEDOUT',
  syscall: 'connect',
  address: '107.170.237.181',
  port: 443 }
```
but i already download sdk in  `.\nw-cache`,why connect 107.170.237.181?

**Comment by adam-lynch** [comment]
Hey @keyiis, I'm not near a Windows machine but I can't reproduce on a Mac. Could this be because you do not have a solid network connection? See #419.

**Comment by piwonesien** [comment]
I had the same issue. I think its a problem for nw-builder with a proxy.  But I have found a few nullpointer, if they are fixed, it's works again (for me): 
https://github.com/piwonesien/nw-builder/commits/master

(but I'm not so familiar with these flavours, so I haven't create a PR for this)

**Comment by adam-lynch** [comment]
Nice find @piwonesien! I've just published 3.2.3 which should fix this. My fix was slightly different (see https://github.com/nwjs-community/nw-builder/commit/44fc918a74b7a961d6cfb16515050fdec334c8aa). Thanks!
